OverlayPlugin.Core/NetworkProcessors/LineDespawnObject.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Runtime.InteropServices;
+using RainbowMage.OverlayPlugin.MemoryProcessors;
+using RainbowMage.OverlayPlugin.NetworkProcessors.PacketHelper;
+
+namespace RainbowMage.OverlayPlugin.NetworkProcessors
```

Comment:
If you've got a couple example lines you could stick in here as comments, that would be super helpful for the cactbot/`LogGuide` updates!